spectacles/client.py

```diff
@@ -349,7 +349,7 @@ def create_query(self, model: str, explore: str, dimensions: List[str]) -> int:
             "*" if len(dimensions) != 1 else dimensions[0],
             query_id,
         )
-        return query_id
+        return result
```

Comment:
Confused how this got past mypy, but you'll need to change the return type hint of this message to match.